src/erocksdb.erl

```diff
@@ -221,8 +221,18 @@ async_open(_CallerRef, _Name, _DBOpts, _CFOpts) ->
                                                      DBOpts::db_options(),
                                                      CFOpts::cf_options()).
 open(Name, DBOpts, CFOpts) ->
+    open(Name, DBOpts, CFOpts, 0).
+
+%% @doc
+%% Open RocksDB with the defalut column family and TTL
+-spec(open(Name, DBOpts, CFOpts, Ttl) ->
+             {ok, db_handle()} | {error, any()} when Name::file:filename_all(),
+                                                     DBOpts::db_options(),
+                                                     CFOpts::cf_options(),
+                                                     Ttl::number()).
```

Comment:
number() is not the appropriate type because which covers both integer() and float(). integer() seems to be suitable according to the corresponding rocksdb function (DBWithTTL::Open) spec.

Reply:
Agree will change this